physionet-django/project/utility.py

```diff
@@ -315,6 +319,8 @@ def grant_gcp_group_access(user, project, data_access):
     email = user.cloud_information.gcp_email.email
     service = create_directory_service(settings.GCP_DELEGATION_EMAIL)
     access = ""
+    # Indicate whether we successfully granted access to the user
+    granted_access = False
```

Comment:
Minor thing but for readability it would be nice if `granted_access = False` wasn't separating `access = ""` from its assignment in the if clause below.

Reply:
More than readability, this needs a fix!